tests/modules/nf-core/falco/main.nf

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { FALCO } from '../../../../modules/nf-core/falco/main.nf'
+
+//
+// Test with single-end data
+//
+workflow test_fastqc_single_end {
```

Comment:
Nitpicking, but shouldn't the test names be changed to `falco`

Reply:
Done